static/css/style.css

```diff
@@ -78,14 +78,19 @@ footer {
     background-color: var(--p-grey);
 }
 
+
+
 footer h4 {
```

Comment:
Would suggest to target with `id` or `class` instead of using `elements` like `h4` if this changes to anything else the CSS breaks.